docs/Reference/ID-Prefix-Reservation.md

```diff
@@ -0,0 +1,154 @@
+---
+# required metadata
+
+title: ID Prefix Reservation Reference | Microsoft Docs
+author: diverdan92
+ms.author: diverdan92
+manager: unniravindranathan
+ms.date: 10/9/2017
+ms.topic: reference
+ms.prod: nuget
+#ms.service:
+ms.technology: null
+#ms.assetid: 
+
+# optional metadata
+
+description: Package ID Prefix Reservation feature description and author guide.
+keywords: NuGet package ID, prefix, reservation
+#ROBOTS:
+#audience:
+#ms.devlang:
+ms.reviewer:
+- ananguar
+- karann-msft
+- unniravindranathan
+#ms.suite:
+#ms.tgt_pltfrm:
+#ms.custom:
+
+---
+
+# Package ID prefix reservation
+[NuGet.org](https://www.nuget.org/) and Visual Studio 2017 version 15.4 or later show a visual indicator for packages that are submitted by owners with a reserved package ID prefix, as long as the package matches the reserved ID prefix naming pattern. The below reference explains what the ID prefix reservation entails, and how an owner can apply for an ID prefix.
+
+## What is package ID prefix reservation?
+Package owners are able to reserve and protect their identity by reserving ID prefixes. Package consumers are provided with additional information when consuming packages that the package they are consuming are not deceptive in their identifying properties. Read on to see how ID prefix reservation works in details.
+
+## ID prefix reservation details 
+When a package ID prefix is reserved, several things happen on the [NuGet.org](https://www.nuget.org/) gallery, as well as in Visual Studio. In addition, there are advanced scenarios that are supported by ID prefix reservations, such as setting a prefix as 'public', delegating prefix subsets to multiple owners.
+
+The below section describes the feature in detail, as well as the more advanced functionality.
+
+### ID prefix reservation on NuGet.org
+When a prefix is reserved on [NuGet.org](https://www.nuget.org/), the following will happen:
+1. A prefix reservation is associated with an owner or set of owners on [NuGet.org](https://www.nuget.org/). 
+2. Whenever a package is submitted to [NuGet.org](https://www.nuget.org/) with an ID that matches the reserved ID prefix, the package is rejected unless it originates from the owner(s) that reserved the ID prefix.
+3. Any package that matches the reserved ID prefix and originates from the owner(s) that reserved the ID prefix will have a visual indicator in Visual Studio 2017 version 15.4 or later, and on [NuGet.org](https://www.nuget.org/) indicating that the package is under a reserved ID prefix. This is true for both new package submissions as well as existing packages under the owner(s). **Note:** The indicator in Visual Studio will only appear if a single feed is selected as the package source. 
+4. All previously existing packages that match the reserved ID prefix, but are *not* owned by the owner of the reserved prefix will remain unchanged (they will not be unlisted, but they will also not have the visual indicator).
```

Comment:
Should we mention that the existing packages can still upload new versions here or somewhere?

Reply:
Updated